### PR TITLE
Update EVerest on kirkstone to 2024.3.0

### PR DESCRIPTION
**Pull request by hikinggrass**

### CODE REVIEW

File: recipes-core/everest/everest-admin-panel_0.2.bb [hunk 1]
```diff
@@ -50,18 +50,17 @@ LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/Apache-2.0;md5=89aea4e17d99a7ca
                     file://node_modules/vuetify/LICENSE.md;md5=61626e12e39ac039026fd688eb9b1b56 \
                     file://node_modules/vuex/LICENSE;md5=761fb17e53b4b87a0507c26d4bd5bfa4"
 
-SRC_URI = "https://github.com/EVerest/everest-admin-panel/releases/download/v0.1.0/everest-admin-panel.tar.gz \
+# TODO: update shrinkwrap file
```
Comment (a-w50) [inline recipes-core/everest/everest-admin-panel_0.2.bb]: Can't this be done in this PR?

File: recipes-support/sqlite/sqlite3_%.bbappend [hunk 2]
```diff
@@ -0,0 +1 @@
+CFLAGS:append = " -DSQLITE_ENABLE_DBSTAT_VTAB"
```
Comment (a-w50) [inline recipes-support/sqlite/sqlite3_%.bbappend:3]: What is this for?  Maybe a small comment would help here.

File: recipes-core/everest/everest-admin-panel_0.2.bb (hunk 1 above)
Comment (hikinggrass) [reply recipes-core/everest/everest-admin-panel_0.2.bb]: yes, I've just updated the shrinkwrap and the license listing

File: recipes-support/sqlite/sqlite3_%.bbappend (hunk 2 above)
Comment (hikinggrass) [reply recipes-support/sqlite/sqlite3_%.bbappend:3]: I've added a comment explaining that this is needed for the auth cache in libocpp for OCPP 2.0.1

**Review by a-w50 — APPROVED** [review]

**Pull request merged**